### ISSUE TITLE
release a new version

### ISSUE THREAD

**Issue by ZoranDuan** [icsharpcode/SharpZipLib#292]
Dear author, 

could you please release another latest version to solve the AES256 encryption compression problem。

thanks a lot。

**Comment by ZoranDuan** [comment]
I can download it on nuget

**Comment by martinweihrauch** [comment]
Yes, this would be great!! I just tried to build the solution from this repository and there are errors ocurring during the build-process. I would also love to have this AES fixed.
Thx
